### ISSUE TITLE
Error on https://anfoz.com/demo-5/

### ISSUE THREAD

**Issue by unblocked-games-premium** [ruffle-rs/ruffle#6686]
# Error Info
Error name: Error
Error message: panicked at 'called `Option::unwrap()` on a `None` value', core/src/display_object/container.rs:853:18
Error stack:
```
Error: panicked at 'called `Option::unwrap()` on a `None` value', core/src/display_object/container.rs:853:18
    at c.wbg.__wbg_new_df6e6ab7a65c4c4d (https://unpkg.com/@ruffle-rs/ruffle/core.ruffle.5ec33e10aba84549cb01.js:1:7087)
    at std::sync::once::Once::call_once::{{closure}}::h9f25555b76706740 (https://unpkg.com/@ruffle-rs/ruffle@0.1.0-nightly.2022.4.11/8b91845f19d7718766e1.wasm:wasm-function[1004]:0x2abe3b)
    at std::sync::once::Once::call_inner::h07d0245bef259c27 (https://unpkg.com/@ruffle-rs/ruffle@0.1.0-nightly.2022.4.11/8b91845f19d7718766e1.wasm:wasm-function[1442]:0x321915)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::h51a4a6ecbb3638ab (https://unpkg.com/@ruffle-rs/ruffle@0.1.0-nightly.2022.4.11/8b91845f19d7718766e1.wasm:wasm-function[4907]:0x47471a)
    at std::panicking::rust_panic_with_hook::h623ac32ff431b114 (https://unpkg.com/@ruffle-rs/ruffle@0.1.0-nightly.2022.4.11/8b91845f19d7718766e1.wasm:wasm-function[3243]:0x4240ab)
    at core::panicking::panic_fmt::h5118e89563022e7e (https://unpkg.com/@ruffle-rs/ruffle@0.1.0-nightly.2022.4.11/8b91845f19d7718766e1.wasm:wasm-function[4980]:0x47850c)
    at core::panicking::panic::he88f1b1582b09c8c (https://unpkg.com/@ruffle-rs/ruffle@0.1.0-nightly.2022.4.11/8b91845f19d7718766e1.wasm:wasm-function[4905]:0x4742ad)
    at core::ops::function::FnOnce::call_once::had0a65b50c8437cf (https://unpkg.com/@ruffle-rs/ruffle@0.1.0-nightly.2022.4.11/8b91845f19d7718766e1.wasm:wasm-function[1089]:0x2c2cae)
    at ruffle_core::avm1::function::Executable::exec::h7be7ee7d4c93a0fb (https://unpkg.com/@ruffle-rs/ruffle@0.1.0-nightly.2022.4.11/8b91845f19d7718766e1.wasm:wasm-function[394]:0x1345ea)
    at <ruffle_core::avm1::object::Object as ruffle_core::avm1::object::TObject>::call_method::ha34611cbebcd022e (https://unpkg.com/@ruffle-rs/ruffle@0.1.0-nightly.2022.4.11/8b91845f19d7718766e1.wasm:wasm-function[360]:0xd3f79)
```

# Player Info
Allows script access: false

# Page Info
Page URL: https://anfoz.com/demo-5/
SWF URL: https://unblocked-games-school.github.io/electricmans/electricman-2-hs.swf

# Browser Info
Useragent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.75 Safari/537.36
OS: Win32

# Ruffle Info
Version: 0.1.0
Name: nightly 2022-04-11
Channel: nightly
Built: 2022-04-11T00:15:18.545Z
Commit: 3d4d7a0ab511377721d5f2c0599230c9dd8d09f5
Is extension: false

**Comment by relrelb** [comment]
Duplicate of #4932.
